### DISCUSSION TITLE
Use numeric team ID instead of abbreviation

### DISCUSSION THREAD

**Discussion by MBKlatt** [vasqued2/ha-teamtracker#198] [Q&A] [answered]
First of all: Sorry for any technical mistakes. I am brand new at github and HA. And thank you for this awesome integration :)

I am trying to retrieve information for my team in Argentina, River Plate.

I used the integration and completed the form like this:

**League:** XXX
**Team:** RIV
**Friendly Name:** River_Plate_LEAGUE
**Conference Number:** 

**Sport Path:** soccer
**League Path:** arg.1

When using this configuration, I am receiving information from the ESPN api, but it corresponds to Independiente Rivadavia, which has the same abbreviation as River Plate (RIV). 

Is there a way I can make the call to the API by using the team's numeric ID (16) insted of the abbreviation? I took a look at the code but I dont quite understand where's the call to the API.

Also, I tried to retrieve information for other teams in Argentina, and I got mixed results. For example, I got the information of all the teams in arg.1 league: https://site.api.espn.com/apis/site/v2/sports/soccer/arg.1/teams and it says Racing abbreviation is "RAC" but when I set up the sensor, I get "NOT_FOUND"

**EDIT:**  Regarding getting mixed results for other teams, it seems it's a timing issue, because only the last game is being returned by the API, which with the configuration mentioned above, I guess it should be: https://site.api.espn.com/apis/site/v2/sports/soccer/arg.1/scoreboard

However, I still don't know how to get around the issue with 2 teams having the same abbreviation. I tried to return Independiente Rivadavia's information by using its numeric ID but it didn't work. I tried both using the team tracker integration and putting the ID number as Team name, and also setting the sensor manually in a YAML file:

```
sensor:
  - platform: teamtracker
    league_id: "XXX"
    sport_path: "soccer"
    league_path: "ARG.1"
    team_id: 694548
    name: "TEST"
```

Thank you so much in advance,

**Comment by vasqued2** [comment, marked as answer]
The team ID is 16.  Looks like the season is over and it doesn't start up again until May?  If so, that's why you aren't getting any results.

You can set up a separate sensor to get their friendly on the 17th.  Use `club.friendly` as the league_path with team_id of 16.  You will still have to wait until a couple days before the match to confirm but it's better than May.

**Reply by MBKlatt** [reply]
Hi David,

Thank you for getting back to me so fast, and happy new year!

I checked the club.friendly API endpoint and saw there's a match scheduled for Ajax in the next days, so I tried completing the team_id with its numerical ID (139) and it worked just fine. I'll wait until River Plate's match is loaded there and see how it goes.

Thank you again!
